docs/INSTALL-devel-flowchart.txt

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
I'd rather not have to re-typeset ASCII art as part of the search/replace :-)

Also, we already do refer to HIL in certain parts (like the repo name).